lib/spack/spack/cmd/buildcache.py

```diff
@@ -58,7 +58,7 @@ def setup_parser(subparser):
                               " tarballs for testing")
     install.add_argument(
         'packages', nargs=argparse.REMAINDER,
-        help="specs of packages to install biuldache for")
+        help="specs of packages to install buildcache for")
```

Comment:
Speaking of typo: `buildache` should probably enter into the vocabulary of people working in deployment of scientific software stacks :laughing: